Sources/SignHereLibrary/Models/Profiles/ProfileType.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ProfileType.swift
+//  Models
+//
+//  Created by Caleb Davis on 06/21/23.
+//
+
+import ArgumentParser
+import Foundation
+
+internal enum ProfileType: String, Codable, ExpressibleByArgument {
```

Comment:
Same comment about including a link to the relevant documentation: https://developer.apple.com/documentation/appstoreconnectapi/profilecreaterequest/data/attributes